ncbi-datasets/13.31.0/README.md

```diff
@@ -0,0 +1,20 @@
+# NCBI datasets and dataformat container
+
+Main tool : [datasets](https://www.ncbi.nlm.nih.gov/datasets/docs/v1/download-and-install/#use-the-datasets-tool-to-download-biological-data) and [dataformat](https://www.ncbi.nlm.nih.gov/datasets/docs/v1/download-and-install/#use-the-dataformat-tool-to-convert-data-reports-to-other-formats)
+
+Full documentation: [https://www.ncbi.nlm.nih.gov/datasets/docs/v1/how-tos/](https://www.ncbi.nlm.nih.gov/datasets/docs/v1/how-tos/)
+
+| Use NCBI Datasets to gather metadata, download data packages, view reports and more
```

Comment:
accidental pipe here? If intending to quote, use a `>` prior to the text

> like so

Reply:
definitely an accident. Sorry!